jobs/payment-jobs/tasks/ap_task.py

```diff
@@ -59,9 +63,58 @@ def create_ap_files(cls):
         6. Create AP file and upload to SFTP.
         7. After some time, a feedback file will arrive and Payment-reconciliation queue will set disbursement
            status to COMPLETED or REVERSED (filter out)
+        8. Find all EFT refunds with status REFUND_REQUESTED.
+        9. Create AP file and upload to SFTP.
+        10. After some time, a feedback file will arrive and Payment-reconciliation queue will move these
+            EFT refunds to REFUNDED. (filter out)
         """
         cls._create_routing_slip_refund_file()
         cls._create_non_gov_disbursement_file()
+        cls._create_eft_refund_file()
+
+    @classmethod
+    def _create_eft_refund_file(cls):
+        """Create AP file for EFT refunds and upload to CGI."""
+        cls.ap_type = EjvFileType.EFT_REFUND
+        eft_refunds_dao: List[tuple[InvoiceModel, EFTRefundModel]] = db.session.query(InvoiceModel, EFTRefundModel) \
+            .join(EFTCreditInvoiceLinkModel, EFTCreditInvoiceLinkModel.invoice_id == InvoiceModel.id) \
+            .join(EFTCreditModel, EFTCreditModel.id == EFTCreditInvoiceLinkModel.eft_credit_id) \
+            .join(EFTShortnameLinksModel, EFTShortnameLinksModel.eft_short_name_id == EFTCreditModel.short_name_id) \
+            .join(EFTRefundModel, EFTRefundModel.short_name_id == EFTShortnameLinksModel.eft_short_name_id) \
+            .filter(EFTRefundModel.status == InvoiceStatus.REFUND_REQUESTED.value) \
+            .filter(EFTRefundModel.refund_amount > 0) \
+            .all()
+
+        current_app.logger.info(f'Found {len(eft_refunds_dao)} to refund.')
+
+        for refunds in list(batched(eft_refunds_dao, 250)):
+            ejv_file_model = EjvFileModel(
+                file_type=cls.ap_type.value,
+                file_ref=cls.get_file_name(),
+                disbursement_status_code=DisbursementStatus.UPLOADED.value
+            ).flush()
+
+            batch_number: str = cls.get_batch_number(ejv_file_model.id)
+            ap_content: str = cls.get_batch_header(batch_number)
+            batch_total = 0
+            line_count_total: int = 0
```

Comment:
prob don't need int, = 0 already indicates? I think you could do 0.0 for float, Decimal('0') for decimal